src/Utils/localImageHelpers.ts

```diff
@@ -0,0 +1,125 @@
+import localforage from "localforage"
+import { useEffect, useState } from "react"
+
+// Expiration time is 5 minutes
+// TODO: Decrease number
+const EXPIRATION_TIME = 5 * 60 * 1000
+const IMAGE_KEY_PREFIX = "IMAGES"
+export const PROFILE_IMAGE_KEY = "PROFILE_IMAGE"
+const DEFAULT_IMAGE_VERSION = "large"
+
+export interface LocalImage {
+  data: string
+  width: number
+  height: number
+  expires?: string
+  photoID?: string
+  aspectRatio?: number
+}
+
+export const storeLocalImage = async (key: string, image: LocalImage) => {
+  const expires = new Date().getTime() + EXPIRATION_TIME
+
+  const storeKey = `${IMAGE_KEY_PREFIX}_${key}`
+  const storeValue = prepareImage(image, expires.toString())
+
+  return localforage.setItem(storeKey, storeValue)
+}
+
+export const getLocalImage = async (
+  key: string
+): Promise<LocalImage | null> => {
+  const storeKey = `${IMAGE_KEY_PREFIX}_${key}`
+  const imageJSONString = await localforage.getItem(storeKey)
+
+  if (!imageJSONString) return null
+
+  return JSON.parse(imageJSONString as string)
```

Comment:
**suggestion:** should we maybe here check if the image expired or not and if yes delete it 👀?